### PR TITLE
Rename package to Vapor

### PR DESCRIPTION
**Pull request by wfortin**

### CODE REVIEW

**Comment by pochretien** [comment]
image

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+
 [![Build Status](https://img.shields.io/travis/coveo/styleguide.svg?style=flat-square)](https://travis-ci.org/coveo/styleguide)
 [![dev-dependencies](https://img.shields.io/david/dev/coveo/styleguide.svg?style=flat-square)](https://github.com/coveo/styleguide/blob/master/package.json)
 [![license](http://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/coveo/styleguide/blob/master/LICENSE)
 
-# Coveo UI Styleguide
+# Vapor CSS
 
-Coveo Styleguide is a collection of UI components used in Coveo's Administration Console. All components and their documentation is [available here](http://coveo.github.io/styleguide/)
+Vapor CSS is Coveo's collection of UI components used in Coveo's Administration Console. All components and their documentation is [available here](http://coveo.github.io/styleguide/)
```
Comment (JPLachance) [inline README.md]: Maybe we should change the GitHub pages url? http://coveo.github.io/vapor/ ?

**Comment by JPLachance** [comment]
Out of scope, but can you add Materialize (https://github.com/Dogfalo/materialize) in the Thanks section of the Readme? I forgot to do it.. :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+![logo](https://github.com/wfortin/vapor/blob/master/vapor.png)
```
Comment (pastjean) [inline README.md]: should not be `/wfortin/vapor/**.png` change that to relative path.
```markdown
![logo](/vapor.png)
 
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
+![logo](https://raw.githubusercontent.com/wfortin/vapor/master/vapor.png)
```
Comment (GermainBergeron) [inline README.md]: Still targeting wfortin?

Comment (pastjean) [inline README.md]: "vapor.png" is enough

**Comment by pastjean** [comment]
pas game de changer le banner par ça :D 
bg2

**Pull request merged**